ethcore/sync/Cargo.toml

```diff
@@ -13,17 +13,21 @@ bytes = { package = "parity-bytes", version = "0.1" }
 client-traits = { path = "../client-traits" }
 common-types = { path = "../types" }
 devp2p = { package = "ethcore-network-devp2p", path = "../../util/network-devp2p" }
+engine = { path = "../engine" }
 enum_primitive = "0.1.1"
 ethcore-io = { path = "../../util/io" }
 ethcore-private-tx = { path = "../private-tx" }
+ethereum-forkid = "0.1"
 ethereum-types = "0.8.0"
+ethash-engine = { path = "../engines/ethash" }
```

Comment:
we don't need `engine` and `ethash-engine` here, right?

Reply:
Done